delsolr.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["License.txt", "Rakefile", "README.rdoc"]
   s.test_files = Dir.glob('test/*_test.rb')
 
-  s.add_dependency("faraday", ["~> 0.8.8"])
+  s.add_dependency("faraday", ["~> 0.9.0"])
```

Comment:
Looks like the current version is 0.12.1. Does it need to be locked to 0.9.x? I'm not suggesting that we upgrade, but that might cause gem versions to conflict down the line.
Also, it looks like the version is locked to 0.8.11 in avvo_common gem. Not sure how that'd work with this.
Anyways, something to watch out for.